specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -15,36 +15,25 @@
 - [Alternatives](#alternatives)
 
 ## Summary
-
-<!--
-Short summary of the proposal. It will be used as user-focused
-documentation such as release notes or a (customer facing) development roadmap.
-The tone and content of the `Summary` section should be
-useful for a wide audience.
--->
+At the moment customers can deploy and manage jobs through the VDK CLI tool or through the API's.
+We want to extend on this and create a web frontend for the VDK.
+The frontend will let customers, create,delete deploy and generally managed nad browse their jobs.
+Frontend is written in an extensible format to easily allow other teams to build extensions.
+To achieve this extensibility the frontend will be built in angular.
 
 ## Glossary
-<!--
-Optional section which defines terms and abbreviations used in the rest of the document.
--->
+data-pipelines: This is the npm project of the bootstrapping codebase it contains all the vdk specific frontend code. for more details please see [README.md](/projects/frontend/data-pipelines/README.md)
+shared-components, @vdk/shared: This is a set of components that are used within the data-pipelines project but are generic and could be re-used by other projects [README.md](/projects/frontend/data-pipelines/README.md)
 
 ## Motivation
-<!--
-It tells **why** do we need X?
```

Comment:
I think it's good to keep the comments, so that we are guided by those during populating the VEP, especially in the very early stages.

Reply:
I find the comments useful. I am easily reminded what the section is meant for. For example, VEP 994 on Jupyter notebook integration keeps its comments and focus. I like this approach

Reply:
Added back in.